pkg/defaults/defaults.go

```diff
@@ -155,7 +154,12 @@ func FromConfig(
 			buildSteps = append(buildSteps, initialReleaseStep)
 
 		} else if testStep := rawStep.TestStepConfiguration; testStep != nil {
-			if testStep.OpenshiftInstallerClusterTestConfiguration != nil {
+			if testStep.MultiStageTestConfiguration != nil {
+				var err error
+				if step, err = steps.MultiStageTestStep(*testStep, config, podClient, jobSpec); err != nil {
+					return nil, nil, fmt.Errorf("unable to create end to end test step: %v", err)
```

Comment:
Are we actually calling this `end to end step` in the user-facing documentation? The design doc calls it multi-stage tests, I think we should call it that way even here.

Reply:
No longer relevant because of https://github.com/openshift/ci-tools/pull/100#discussion_r318066351.